### PR TITLE
Makes elder atmosian statue craftable

### LINKED ISSUE #74534: Elder Atmosian statue uncraftable?
Reporting client version: 514.1589

## Round ID:
203201

## Testmerges:
- [Disables auxtools](https://www.github.com/tgstation/tgstation/pull/74497)
- [Clients can't OOC until they have fully gone through `New()`](https://www.github.com/tgstation/tgstation/pull/74493)
- [The North Star Expeditionary Vessel - A Second Wind](https://www.github.com/tgstation/tgstation/pull/74371)
- [Adds Master.sleep_delta to the mc tab.](https://www.github.com/tgstation/tgstation/pull/74360)
- [Imports and Contraband: Different! Cargo crates without locks! MEAT!](https://www.github.com/tgstation/tgstation/pull/74490)

## Reproduction:

The Elder Atmosian statue no longer seems to be in the crafting menu?  Perhaps since the recent UI changes to it?

Or I'm missing a trick, if so, please let me know!

### PR DESCRIPTION
**Pull request by RealityOverseer**
## About The Pull Request
Fixes https://github.com/tgstation/tgstation/issues/74534 by making elder atmosian statue craftable. I am not 100% sure if this is the intended way for it to be able to be made or if it's even the right amount of materials but if I need to change it please let me know.
## Why It's Good For The Game
Fixing issues is good, having uncraftable things is bad.
## Changelog
:cl:Reality Overseer
fix: makes elder atmosian statue craftable
/:cl:

### CODE REVIEW

**Comment by MTandi** [comment]
Put it under CAT_ATMOSPHERIC or CAT_STRUCTURE

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 1]
```diff
@@ -466,6 +466,7 @@ GLOBAL_LIST_INIT(metalhydrogen_recipes, list(
 	new /datum/stack_recipe("ancient armor", /obj/item/clothing/suit/armor/elder_atmosian, req_amount = 5, res_amount = 1, check_density = FALSE, category = CAT_CLOTHING),
 	new /datum/stack_recipe("ancient helmet", /obj/item/clothing/head/helmet/elder_atmosian, req_amount = 3, res_amount = 1, check_density = FALSE, category = CAT_CLOTHING),
 	new /datum/stack_recipe("metallic hydrogen axe", /obj/item/fireaxe/metal_h2_axe, req_amount = 15, res_amount = 1, check_density = FALSE, category = CAT_WEAPON_MELEE),
+	new /datum/stack_recipe("elder atmosian statue", /obj/structure/statue/elder_atmosian, 10, one_per_turf = TRUE, on_solid_ground = TRUE, category = CAT_ENTERTAINMENT),
```
[the remark this answers is not in the document]
Comment (RealityOverseer) [reply code/game/objects/items/stacks/sheets/mineral.dm]: Fixed and tested

**Comment by MTandi** [comment]
```
/datum/crafting_recipe/elder_atmosian_statue
	name = "Elder Atmosian Statue"
	result = /obj/structure/statue/elder_atmosian
	time = 6 SECONDS
	reqs = list(/obj/item/stack/sheet/mineral/metal_hydrogen = 20,
				/obj/item/stack/sheet/mineral/zaukerite = 15,
				/obj/item/stack/sheet/iron = 30,
				)
	category = CAT_STRUCTURE
```
It was originally in recipes.dm, not sure where this file went

**Comment by dragomagol** [comment]
> ```
> /datum/crafting_recipe/elder_atmosian_statue
> 	name = "Elder Atmosian Statue"
> 	result = /obj/structure/statue/elder_atmosian
> 	time = 6 SECONDS
> 	reqs = list(/obj/item/stack/sheet/mineral/metal_hydrogen = 20,
> 				/obj/item/stack/sheet/mineral/zaukerite = 15,
> 				/obj/item/stack/sheet/iron = 30,
> 				)
> 	category = CAT_STRUCTURE
> ```
> 
> It was originally in recipes.dm, seems like it was missed after #72856 . Maybe there was a reason? @dragomagol FYI
> 
> Not sure if we lost any other recipes, needs checking.

That was a mistake, I didn't intend to remove any recipes

**Review by dragomagol — APPROVED** [review]

File: code/datums/components/crafting/atmospheric.dm [hunk 2]
```diff
@@ -335,3 +335,13 @@
...
 	category = CAT_ATMOSPHERIC
+
+/datum/crafting_recipe/elder_atmosian_statue
+	name = "Elder Atmosian Statue"
+	result = /obj/structure/statue/elder_atmosian
+	time = 6 SECONDS
+	reqs = list(/obj/item/stack/sheet/mineral/metal_hydrogen = 20,
+				/obj/item/stack/sheet/mineral/zaukerite = 15,
+				/obj/item/stack/sheet/iron = 30,
+				)
```
Comment (Mothblocks) [inline code/datums/components/crafting/atmospheric.dm:346]:
```suggestion
	reqs = list(
		/obj/item/stack/sheet/mineral/metal_hydrogen = 20,
		/obj/item/stack/sheet/mineral/zaukerite = 15,
		/obj/item/stack/sheet/iron = 30,
	)
```

**Pull request merged**